gradle.properties

```diff
@@ -3,7 +3,7 @@ kotlin.code.style=official
 platformType=IU
 platformVersion=222.4345.14
 platformDownloadSources=true
-platformPlugins = com.intellij.java, org.jetbrains.kotlin, android, maven, gradle, Groovy
+platformPlugins = com.intellij.java, org.jetbrains.kotlin, android, maven, gradle, Groovy, com.jetbrains.php:222.4345.14
```

Comment:
I did not know it, but `gradle.properties` actually [does not support string interpolation](https://stackoverflow.com/questions/23853044/does-gradle-support-properties-interpolation) 😞 

Let's keep it like this for now, but I guess we should move constants to build file or some separate file in the future.